prototypes/entities.lua

```diff
@@ -319,6 +319,9 @@ local function create_inserters(prefix, next_prefix, base_underground_name, tint
   if prefix == "space-" then
     loader_inserter.collision_mask = {"floor-layer", "item-layer", "object-layer", "water-tile"}
   end
+  if prefix == "deep-space-" then
```

Comment:
You can merge these two and use an `or` in the if clause.

Reply:
I agree. My intention was to have as few modifications as possible in existing code, but i agree your proposal is the better/cleaner way to implement this..